router_configuration_backups/br-mdf-01

```diff
@@ -0,0 +1,385 @@
+## Last changed: 2019-10-27 02:59:16 UTC
```

Comment:
can we move these to under `switch-configuration`? Something like `switch-configuration/config/backups`  since I feel liek that would make more sense then the toplevel dir here

Reply:
Well. We can, but it goes against my network admin view that there’s a distinction between switches and routers. Everything under switch-configuration relates to auto-generation of config a for the ex-series switches. The router config are manually maintained at least for now. 

As such, having it separate at top level actually makes more sense to me.

Reply:
Fair enough then can it at least follow the existing naming convention? So `router-configuration/`

O and arent EX-series switches are L2/L3?! :smile:

Reply:
Any thoughts on the dir?